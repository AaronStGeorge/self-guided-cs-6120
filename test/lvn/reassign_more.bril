@main {
  a: int = const 100;
  a: int = add a a;
  a: int = add a a;
  a: int = const 42;
  print a;
}
