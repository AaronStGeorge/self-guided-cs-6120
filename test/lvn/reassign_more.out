@main {
  lvn.0: int = const 100;
  lvn.1: int = add lvn.0 lvn.0;
  lvn.2: int = add lvn.1 lvn.1;
  a: int = const 42;
  print a;
}
